### ISSUE TITLE
Auto disconnect

### ISSUE THREAD

**Issue by ngvietdung96** [washo4evr/Socket.io-v1.x-Library#60]
Hi, after few time with my ESP32 project, when i connect and don't communicate anything with server, after few second it'll disconnect from socket.io server. 

hope you check that problem. Have a nice day.

**Comment by HRahikainen** [comment]
Yes. I would also like to know, if you need to manually send the client.heartbeat() and how to use it. It seems to me that my ESP32 cliient disconnects after the 20-30 seconds that it isn't sending or receiving. I'm not explicitly sending heartbeat from the server (express/nodejs) at the moment. My connection to this hardware piece needs to stay online.

Hope you see this and thanks for your work so far.
